packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -74,7 +82,7 @@ export default util.createRule<Options, MessageIds>({
     const methodCheck = overrides.methods || baseCheck;
     const propCheck = overrides.properties || baseCheck;
     const paramPropCheck = overrides.parameterProperties || baseCheck;
-
+    const exceptMethods = option.exceptMethods || [];
```

Comment:
Considering we're going to repeatedly checking for existence in here, using a `Set` will convert the lookup to `O(1)` rather than `O(n)` which could save a bunch of time on larger configs / codebases.

```suggestion
    const exceptMethods = new Set(option.exceptMethods || []);
```

Reply:
I haven't really used `Set` much, but I think I should use it more(it looks very convenient)